From 701f39a65a5df5c485dc1431046f03ec1455ac94 Mon Sep 17 00:00:00 2001
From: Nicolas Schier <n.schier@avm.de>
Date: Wed, 16 Aug 2023 08:22:19 +0200
Subject: [PATCH 39/58] MAINTAINERS: Add usr/ (initramfs generation) to KBUILD

Add scripts for generating initramfs to KBUILD, to prevent idling of
patches for usr/.

Signed-off-by: Nicolas Schier <n.schier@avm.de>
Signed-off-by: Masahiro Yamada <masahiroy@kernel.org>
---
 MAINTAINERS | 1 +
 1 file changed, 1 insertion(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 4cc6bf79f..9f6217420 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11296,6 +11296,7 @@ F:	scripts/dummy-tools/
 F:	scripts/mk*
 F:	scripts/mod/
 F:	scripts/package/
+F:	usr/
 
 KERNEL HARDENING (not covered by other areas)
 M:	Kees Cook <keescook@chromium.org>
-- 
2.42.0

